PrebidMobile/API1.0/src/test/java/org/prebid/mobile/PrebidServerAdapterTest.java

```diff
@@ -135,7 +144,15 @@ public void testInvalidPrebidServerConfigIdForAppNexusHostedPrebidServer() {
 
     @Test
     public void testInvalidPrebidServerConfigIdForRubiconHostedPrebidServer() {
-        PrebidMobile.setPrebidServerHost(Host.RUBICON);
+
+        if (!successfulMockServerStarted) {
+            fail("Mock server was not started");
+        }
+        server.enqueue(new MockResponse().setResponseCode(400).setBody("Invalid request format: No stored imp found for id: 1001-1_INVALID_CONFIG_ID"));
```

Comment:
Sorry for reiterating on this, but can we have that response body from the `MockPrebidServerResponses`?

Something like: `MockPrebidServerResponses.invalidConfigIdFromRubicon`.